package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk",
   "author": "UMA Team",
-  "version": "3.2.11",
+  "version": "3.2.11-beta.1",
```

Comment:
The only thing I think we should be careful about is this version; I assume this PR will be held until a proper release can be made from the contracts repo.

Reply:
Yes, exactly @pxrl